### DISCUSSION TITLE
reinforceAttack issue

### DISCUSSION THREAD

**Discussion by loayei** [coccoinomane/crabada.py#32] [Q&A] [answered]
Hey, @coccoinomane, for some reason even if I set the strategy highestBp and reinforcement to pick to "1". I am still not getting the usual highest Bp bulks.
My max tus price is set high as well.

Not sure what is happening today.

Maybe cz of the sudden influx of crabs to the tavern, the bot is acting up?

**Comment by loayei** [comment]
Issue seems to be fixed now after tavern pages went down from 280 to 170

**Comment by coccoinomane** [comment, marked as answer]
Fixed in https://github.com/coccoinomane/crabada.py/issues/34
